SocialCareCaseViewerApi/V1/Boundary/Requests/UpdateCaseSubmissionRequest.cs

```diff
@@ -27,6 +27,12 @@ public UpdateCaseSubmissionRequestValidator()
             RuleFor(s => s.EditedBy)
                 .NotNull().WithMessage("Provide who is updating the submission")
                 .EmailAddress().WithMessage("Provide a valid email address for who is updating the submission");
+            When(s => s.Residents != null, () =>
+            {
+                RuleFor(s => s.Residents!.Count).GreaterThan(0)
```

Comment:
`!` tells the compiler that s.Residents isn't null, it's not smart enough to figure this out from the `When` statement unfortunately